src/app/pages/ConnectorDetailsPage/components/ConfigurationTab/CommonStep.tsx

```diff
@@ -112,7 +112,15 @@ export const CommonStep: FC<CommonStepProp> = ({
           isRequired
           fieldId="service-account"
           labelIcon={
-            <Popover bodyContent={<p>{t('serviceAccountDescText')}</p>}>
+            <Popover
+              bodyContent={
```

Comment:
Awesome!  The only last improvement I could suggest here is instead of using `<p>` consider replacing the fragment with `TextContent`  (mentioned [here](https://www.patternfly.org/v4/components/text#textcontent)) and then each `<p>` with `<Text component={'p'}></Text>` (mentioned [here](https://www.patternfly.org/v4/components/text#text)).

I opt to use Text over raw tags mostly for consistency in the code and this also ensures in some cases the correct styling is applied.  It may not however actually make any difference in this particular case, maybe give it a try and see.  If it makes no difference I think we can leave it for now.

Reply:
Yes, it does make difference by using <Text component={'p'}></Text>. Font size will change from 14px to 16px, if we replace <p> tag from <Text>. In our application all the help text have font size of 14px. 
Other option is using <Text component={TextVariants.small}></Text>. But this needs color poperty to be changed. Should I do that?

Reply:
Yep, I think that's probably the way.  You could use `className` and pass one of [these modifiers](https://www.patternfly.org/v4/utilities/text#standard-colors) *or* another option I've used is to import the color value from `@patternfly/react-tokens` and use that, kind of [like this](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx#L22) and then it's used later on in the file [like so](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx#L58).